client/layout/index.jsx

```diff
@@ -184,7 +184,10 @@ class Layout extends Component {
 				{ this.props.isOffline && <OfflineStatus /> }
 				<div id="content" className="layout__content">
 					{ config.isEnabled( 'jitms' ) && this.props.isEligibleForJITM && (
-						<AsyncLoad require="calypso/blocks/jitm" messagePathSuffix="admin_notices" />
+						<AsyncLoad
+							require="calypso/blocks/jitm"
+							messagePath={ `calypso:${ this.props.sectionJitmPath }:admin_notices` }
+						/>
```

Comment:
Can you please add `placeholder={ null }` here? Somehow I forgot to add it in #46770 🙁